website/content/docs/connect/config-entries/mesh.mdx

```diff
@@ -365,6 +442,21 @@ Note that the Kubernetes example does not include a `partition` field. Configura
         },
       ],
     },
+    {
+      name: 'Peering',
+      type: 'PeeringMeshConfig: <optional>',
+      description:
+        'Controls configuration specific to [peering connections](/docs/connect/cluster-peering).',
+      children: [
+        {
+          name: 'PeerThroughMeshGateways',
+          type: 'bool: <optional>',
+          description: `Determines if peering control-plane traffic should be routed through mesh gateways.
+                        When enabled, dialing cluster will attempt to contact peers through their mesh gateway.
+                        Accepting clusters will advertise the address of their mesh gateways as opposed to their Consul servers directly.`,
```

Comment:
```suggestion
                        When enabled, dialing cluster attempt to contact peers through their mesh gateway.
                        Clusters that accept calls advertise the address of their mesh gateways, rather than the address of their Consul servers.`,
```